test/plugins/test_mikrotik_scan.py

```
@@ -0,0 +1,83 @@
+
+def test_disabled_lease_without_mac_does_not_abort_remaining_leases():
+    leases = [
+        _lease("*1", "192.168.1.2", "aa:bb:cc:dd:ee:01"),
+        _lease("*2", "192.168.1.5", None, status="waiting"),
+        _lease("*3", "192.168.1.8", "aa:bb:cc:dd:ee:03"),
+    ]
+    api = MagicMock(return_value=leases)
+    plugin_objects = MagicMock()
+
+    mikrotik.MT_USER = "user"
+    mikrotik.MT_PASS = "password"
```

Ruff (0.16.2)

[error] Possible hardcoded password assigned to: "MT_PASS"

(S105)